src/main/scala/DeathRun.scala

```diff
@@ -0,0 +1,77 @@
+class DeathRun {
+var overallDead = false
+  def choices(Chara:Character, deathf:Boolean = false,deatha:Boolean = false, deaths:Boolean = false, deathm:Boolean = false,success:String = "somehow it worked"):String={
+    //player chooses 1 of 4 actions.
+    var choice = readLine("[Fight,Dodge,Run,Cast Magic]").toLowerCase
+    choice.substring(1,1) match {
+      case "f" =>
+        if (deathf | Chara.attributes.strength < 4){
+          this.overallDead = true
+          return "You died"
+        }
+        else{
+          return success
+        }
+      case "d" =>
+        if (deatha| Chara.attributes.agility < 4){
+          this.overallDead = true
+          return "You died"
+        }
+        else{
+          return success
+        }
+      case "r" =>
+        if(deaths | Chara.attributes.speed < 4){
+          this.overallDead = true
+          return "You died"
+        }
+        else{
+          return success
+        }
+      case "c" =>
+        if(deathm | Chara.attributes.magic == 0){
+          this.overallDead = true
+          return "You died"
+        }
+        else{
+          Chara.attributes.magic -= 1
+          return success
+        }
+    }
+  }
+}
+class Jungle{
+
+}
+class Volcano{
+
+}
+class Beach{
```

Comment:
usually all the cllasses would be in their own separate files, otherwise it;s hard to keep trach of what's where